.s2i/bin/assemble

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+yum install postgresql-96
+/usr/libexec/s2i/assemble
```

Comment:
```bash
yum install -y postgresql-96
exec /usr/libexec/s2i/assemble
```

... but as you noted down bellow, this will not work

Reply:
dont install stuff during runtime!

Reply:
Thanks for the feedback :smile: